### PR TITLE
Added get MinNotional & possible fix for 24 hours wsi-timeout scheduling validity check

### PR DESCRIPTION
**Pull request by mussonero**
Added get MinNotional and possible fix for 24 hours wsi-timeout scheduling validity check.

### CODE REVIEW

File: src/binance_websocket.cpp [hunk 1]
```diff
@@ -142,15 +142,17 @@ void binance::Websocket::connect_endpoint(CB cb, const char* path)
...
-	}
+    auto start = std::chrono::high_resolution_clock::now();
+    auto end = start + hours;
+    do {
+        using namespace std::chrono;
+
+        lws_service(context, 500);
+        if (lws_service_cancelled)
+            break;
+    } while (std::chrono::high_resolution_clock::now() < end);
```
Comment (dmikushin) [inline src/binance_websocket.cpp:155]: High-resolution clock is maybe too precise for this situation? Low-resolution clock would put less workload on the system

File: src/binance_market.cpp [hunk 2]
```diff
@@ -177,6 +177,41 @@ binanceError_t binance::Market::getTickSize(const char *symbol, double& maxPrice
...
+                        status = binanceSuccess;
+                        break;
+                    }
+
+                }
+            }
+        }
+    }
+
+    Logger::write_log("<get_lotSize> Done.");
```
Comment (dmikushin) [inline src/binance_market.cpp:210]: Logger message should be different here

File: src/binance_market.cpp [hunk 3]
```diff
@@ -177,6 +177,41 @@ binanceError_t binance::Market::getTickSize(const char *symbol, double& maxPrice
 	return status;
 }
 
+binanceError_t binance::Market::getMinNotional(const char *symbol, double& minNotional)
```
Comment (dmikushin) [inline src/binance_market.cpp:180]: Excellent function! Let's make this into a separate commit?

File: src/binance_websocket.cpp (hunk 1 above)
Comment (mussonero) [reply src/binance_websocket.cpp:155]: Yes I agree on using Low-resolution clock, the system will benefit from it.

File: src/binance_market.cpp (hunk 3 above)
Comment (mussonero) [reply src/binance_market.cpp:180]: thanks! :)

**Comment by mussonero** [comment]
> But why should not we check ws for an empty response and restart automatically if it is the case? - This sounds to me like a more straight-forward logic.

I agree again with you regarding `checking wsi for an empty response` .
But at the moment, i setup in my program to restart automatically when `websocket::enter_event_loop()` exiting the loop, and it seems works fine for now :) .

**Comment by dmikushin** [comment]
Thanks a lot, let's move on for now - we can adjust the event loop behavior later.
